src/cpp/rtps/resources/TimedEventImpl.h

```diff
@@ -118,7 +84,8 @@ class TimedEventImpl
      * @param time_millisec New expiration time in milliseconds.
      * @return true on success
      */
-    bool update_interval_millisec(double time_millisec);
+    bool update_interval_millisec(
+            double time_millisec);
```

Comment:
Rename param to `interval` (in the doc also). The doc mentions `restart_timer`, which does not exist on this class.